packages/grafana-data/src/types/panel.ts

```diff
@@ -17,10 +17,15 @@ export interface PanelPluginMeta extends PluginMeta {
 
 export interface PanelData {
   state: LoadingState;
+  /**
+   * Contains data frames with field overrides applied
+   */
```

Comment:
for these property docs I prefer the oneliner comment

```typescript
export interface ApplyFieldOverrideOptions {
  /* Contains data frames with field overrides applied  */
  data?: DataFrame[];

  /* blah blah blash  */
  fieldOptions: FieldConfigSource;

  /* It replaces them with candy!  */
  replaceVariables: InterpolateFunction;

  /* Contains data frames with field overrides applied  */
  theme: GrafanaTheme;

  /* Contains data frames with field overrides applied  */ 
  timeZone?: TimeZone;

  /* Contains data frames with field overrides applied  */
  autoMinMax?: boolean;

  /* Contains data frames with field overrides applied  */
  standard?: FieldConfigEditorRegistry;

  /* Contains data frames with field overrides applied  */
  custom?: FieldConfigEditorRegistry;
}
```

Reply:
Single line comment's won't work with docs generator unfortunately :/

Reply:
or is it '//' comments only that does not work with docs generator @mckn ?

Reply:
oh, strange thought it was part of the standard 

But seems it must start with two stars 
https://api-extractor.com/pages/tsdoc/doc_comment_syntax/

Reply:
Yeah, so // will be considered as code comments. And /** */ will be considered as code comments that should be part of the documentation. If that makes sense?